### ISSUE TITLE
Delay before Live as-you-type conversion

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#117]
The live as-you-type conversion is great and I use it, but when you deal with 800 lines of code of source code, even minor changes are terribly slow, since it takes even 10 seconds to register a single key stroke.

What would help a lot would be if the conversion was deferred by a second after the last keystroke - i.e. don't start conversion until the user hasn't been typing for 1 second.

**Comment by drkameleon** [comment]
Well, what can I say. This is a very important topic.

Let me share 1-or-2 details about the inner workings:

- I know it is a bit slow when working with somewhat bigger files
- Making conversion on-demand (as it is right now, via the Preferences) is an option. But it's not what I'd definitely want.
- What you suggest makes sense. However, I still consider it as an "intermediate" solution.
- I've already found a way to make it 10x as fast (seriously!) though I'm still having some development issues (not having anything to do with the app itself, but mostly with the embedded... Licensing - sounds rather weird, but true)

**Comment by drkameleon** [comment]
Feature implemented as of the upcoming 2.1.

![screen shot 2015-11-30 at 1 49 43 pm](https://cloud.githubusercontent.com/assets/1265028/11471957/4850c05a-9769-11e5-865e-2994ab907a40.png)
